modules/util/css-builder/src/main/java/com/liferay/css/builder/maven/BuildCSSMojo.java

```diff
@@ -71,7 +71,7 @@ public void setDirNames(String dirNames) {
 	}
 
 	/**
-	 * @parameter
+	 * @parameter default-value="${project.build.directory}/${project.build.finalName}"
```

Comment:
@david-truong can we make the `css.builder` plugin configuration sections simply now in the archetypes?